client/Gruntfile.js

```diff
@@ -40,50 +39,46 @@ module.exports = function (grunt) {
         'template:docs',
         'connect:test',
         'open:docs',
-        'ngtemplates',
+        'ngtemplates:docs',
         'watch'
     ]);
 
     grunt.registerTask('server', [
         'clean',
-        'style',
-        'template:test',
-        'connect:test',
-        'open:test',
-        'ngtemplates',
-        'watch'
+        'copy:assets',
+        'copy:index',
+        'copy:sirTrevor',
+        'ngtemplates:dev',
+        'webpack-dev-server:start'
     ]);
+
+    grunt.registerTask('ci:travis', []);
+    //grunt.registerTask('ci:travis', [
```

Comment:
About this one, all the linting should go in there. But I think in the near future, we'll just use the superdesk eslint configuration